grpc/agent/agent_unix.go

```diff
@@ -192,3 +167,320 @@ func GetSystemDependencies() []string {
 		"Requires=network.target",
 		"After=network-online.target"}
 }
+
+// Validation functions
+func GetNetBanner(portnum int64) (bool, error) { // exists (boolean)
+	return true, nil
+}
+
+func NetHttpContentRegex(full_url string) (string, error) { // content hash (string)
+	net_resp, err := http.Get(full_url)
+	if err != nil {
+		return "", err
+	}
+	defer net_resp.Body.Close()
+	page_html, deserialize_err := ioutil.ReadAll(net_resp.Body)
+	if deserialize_err != nil {
+		return "", deserialize_err
+	}
+
+	// return MD5Sum(fmt.Sprintf("%s", page_html)), nil
+	return string(page_html[:]), nil // stringify
+}
```

Comment:
This function doesn't do any regex anything. It just gets the content and returns it. The agent.go function doesn't do regex either?